ios/Classes/SwiftFpjsProPlugin.swift

```diff
@@ -56,7 +56,7 @@ public class SwiftFpjsProPlugin: NSObject, FlutterPlugin {
         return metadata
     }
 
-    private func parseRegion(passedRegion: String?, endpoint: String?) -> Region {
+    private func parseRegion(passedRegion: String?, endpoint: String?, endpointFallbacks: [String] = []) -> Region {
```

Comment:
The default value of the `endpointFallbacks` parameter does not seem to be used anywhere in the code. In line 22, there is already a fallback to an empty array if the cast fails.

Reply:
Thank you, fixed!